Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -41,6 +42,10 @@ class PlaceCarouselViewController: UIViewController {
         didSet {
             // TODO: how do we make sure the user wasn't interacting?
             placeCarousel.refresh()
+
+            if oldValue.count == 0 {
```

Comment:
I'm a little concerned that we might hit an edge case with this but it's good enough for now. :)